cocotb/share/lib/fli/FliImpl.h

```diff
@@ -343,6 +343,11 @@ class FliStringObjHdl : public FliValueObjHdl {
     char *m_mti_buff = nullptr;
 };
 
+/** Maintains a cache of FliTimedCbHdl objects which can be reused.
+ *
+ * Apparently allocating and freeing Timer callback objects is very expensive
```

Comment:
Too bad we don't know why it was added :( Shows even more why commit messages are important (https://github.com/cocotb/cocotb/commit/ff68e85d3863c492a2189785731716f4bb257d6f)